olive/passes/onnx/conversion.py

```diff
@@ -449,3 +456,34 @@ def _convert_distributed_model_on_device(
             model_name_pattern=DistributedOnnxModel.DEFAULT_RANKED_MODEL_NAME_FORMAT,
             num_ranks=world_size,
         )
+
+
+class OnnxOpVersionConversion(Pass):
+    @staticmethod
+    def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
+        latest_opset_version = onnx.defs.onnx_opset_version()
+
+        config = {
+            "target_opset": PassConfigParam(
+                type_=int,
+                default_value=latest_opset_version,
+                description="The version of the default (ai.onnx) opset to target. Default: latest opset version.",
+            ),
+        }
+        config.update(get_external_data_config())
+        return config
+
+    def _run_for_config(
+        self, model: ONNXModelBase, data_root: str, config: Dict[str, Any], output_model_path: str
```

Comment:
Looks like the both the input and output model types are just `ONNXModel` like most of the other onnx passes? 
The composite and distributed cases are handled by https://github.com/microsoft/Olive/blob/main/olive/passes/olive_pass.py#L346

This pass only works with `ONNXModel` type.

Reply:
Removed the check.